server/test/handlers/account.test.js

```diff
@@ -129,7 +129,7 @@ describe('Test to check valid account creation', () => {
             const user = await accountHandler.createUserAccount(firstName, lastName, type, studentId, email, password, phone, userType, workerId, specialization, schoolId) ;
             
             // Assert
-            expect(user).toEqual(expect.objectContaining(workerUser));
+            expect(user).toEqual(expect.objectContaining(user));
```

Comment:
Hi Nancy, 
I would recommend reverting this test code as well to avoid having x = x and ensure proper testing.

Reply:
Hi Amy,
Good catch!
My Action: I have reverted the incorrect commit and replaced line 97 with `expect(user).toEqual(expect.objectContaining(workerUser));`

You can see this change in line 132 of server/test/handlers/account.test.js

Please confirm if this takes care of your suggestion!